src/Pos/Wallet/Web/ClientTypes.hs

```diff
@@ -161,21 +161,22 @@ convertTxOutputs :: [TxOut] -> [(CId w, CCoin)]
 convertTxOutputs = map (addressToCId . txOutAddress &&& mkCCoin . txOutValue)
 
 -- | Get all addresses of source account of given transaction.
-getLocalAccountAddrs
-    :: [CWAddressMeta]  -- ^ all addresses in wallet
-    -> [CId Addr]       -- ^ Input addresses of transaction
-    -> Either Text [CId Addr]
-getLocalAccountAddrs walAddrMetas inputAddrs = do
+getChangeAccountAddrs
```

Comment:
Perhaps a name like `getTxChangeAddresses` would be more appropriate? I don't see why it should mention `Account`.